modules/ob-authentication-to-container-registries.adoc

```diff
@@ -0,0 +1,47 @@
+// This module is included in the following assembly:
+//
+// builds/authenticating-build-images.adoc
+
+:_content-type: PROCEDURE
+[id="ob-authentication-to-container-registries_{context}"]
+= Authentication to container registries
+
+To push images to a private container registry, you must define a secret in the `OpenShift-Builds` namespace and then reference it in your `Build` custom resource (CR).
+
+
+.Procedure
+. Run the following command to generate a secret:
++
+[source,terminal]
+----
+oc --namespace <YOUR_NAMESPACE> create secret docker-registry <CONTAINER_REGISTRY_SECRET_NAME> \
+  --docker-server=<REGISTRY_HOST> \ <1>
+  --docker-username=<USERNAME> \ <2>
+  --docker-password=<PASSWORD> \ <3>
+  --docker-email=me@here.com
```

Comment:
User-replaceable values should be lowercase with underscores, as below:
```suggestion
oc --namespace <namespace> create secret docker-registry <container_registry_secret_name> \
  --docker-server=<registry_host> \ <1>
  --docker-username=<username> \ <2>
  --docker-password=<password> \ <3>
  --docker-email=<email_address>
```
In this block, the email should be a user-replaceable value as well. Double-check the files in this PR to make sure they all have this format (I noticed some later with spaces instead of underscores).